apps/frontend/src/app/data-stores/component/data-stores.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { DataStoresService } from '../service/data-stores-service.service';
+import { Subject, takeUntil } from 'rxjs';
+import { DataStore } from '../../shared/types/data-store';
+
+@Component({
+  selector: 'app-data-stores',
+  templateUrl: './data-stores.component.html',
+  styleUrl: './data-stores.component.css',
+})
+export class DataStoresComponent implements OnInit, OnDestroy {
+  private readonly destroy$ = new Subject<void>();
+  dataStores: DataStore[] = [];
```

Comment:
Please use observables or subjects when handling data from the backend. Observables provide real-time responsiveness. When the data from the dataStoresService changes, our component will automatically receive the updated data without you having to manually refresh or poll for changes. With a regular array, you'd only get the initial state.